getting-started/gopher-run/cmd/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"html"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"cloud.google.com/go/firestore"
+	"google.golang.org/api/iterator"
+)
+
+func handlePost(w http.ResponseWriter, r *http.Request) {
+	//Open database
+	projectID := "maralder-start"
+	ctx := context.Background()
+	client, err := firestore.NewClient(ctx, projectID)
+	if err != nil {
+		log.Fatalf("Error creating client, %v", err)
```

Comment:
Error messages are usually the name of the function that failed followed by the error, like this:
```suggestion
		log.Fatalf("firestore.NewClient: %v", err)
```

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors

Reply:
Done